olive/model/handler/mixin/hf.py

```diff
@@ -84,18 +91,25 @@ def save_metadata(self, output_dir: str, exclude_load_keys: Optional[List[str]]
             )
             saved_filepaths.extend(module_files)
         save_model_config(config, output_dir, **kwargs)
-        saved_filepaths.append(str(output_dir / "config.json"))
+        saved_filepaths.append(str(config_file_path))
```

Comment:
Just flying by: prefer https://docs.python.org/3/library/os.html#os.fspath to convert a pathlib path to string

Reply:
thanks for the info. I will look into this. We do the pathlib.Path to string conversion in multiple places so will have to update it in many places if we decide to follow this convention